http/src/main/resources/swagger/api-docs.yaml

```diff
@@ -987,6 +987,82 @@ definitions:
           type: string
         description: The scopes for the cluster.
 
+  ListClusterResponse:
```

Comment:
the only difference is removing `errors` field which isn't used by anyone. And it's value was misleading